src/Tasks/Microsoft.NET.Build.Tasks/sdk/Sdk.props

```diff
@@ -74,6 +74,12 @@ Copyright (c) .NET Foundation. All rights reserved.
     <BaseIntermediateOutputPath Condition="'$(BaseIntermediateOutputPath)' == ''">$(ArtifactsPath)\obj\</BaseIntermediateOutputPath>
   </PropertyGroup>
 
+  <!-- Record whether ArtifactsPath / UseArtifactsOutput was set at this point in evaluation.  We will generate an error if these properties are set
```

Comment:
Should this be inside the PropertyGroup on line 67 (it has the same condition)?

Reply:
Good idea, but I will try to do that later since I'd like to get this in before we branch for Preview 4.